projects/core/src/configurator/commons/facade/configurator-groups.service.ts

```diff
@@ -130,21 +192,25 @@ export class ConfiguratorGroupsService {
    *
    * @param configuration - Configuration
    * @param groupId - Group ID
+   * @param doNotSetStatus - Group status will not be set for previous group
    */
   public navigateToGroup(
     configuration: Configurator.Configuration,
-    groupId: string
+    groupId: string,
+    doNotSetStatus?: boolean
```

Comment:
What about **isStatusSet** instead of **doNotSetStatus**? Is **isStatusSet** not more readable?

Reply:
I named it setStatus and added a default value.
I named it doNotSetStatus because I wanted to make it optional without changing the existing method caller.